event_dialog.py

```diff
@@ -51,7 +60,7 @@ def ewkb_to_geom(ewkb_str):
         header = header[:6] + "%X" % (int(header[6], 16) ^ 2) + header[7]
         # remove srid
         ewkb_str = ewkb_str[:2] + header + ewkb_str[18:]
-    w = ewkb_str.decode('hex')
+    w = binascii.unhexlify(ewkb_str)
```

Comment:
Isn't `bytes.fromhex(ewkb_str)` equivalent ? (but without an import)

Reply:
done in 5b53f09